spec/models/activity_spec.rb

```diff
@@ -1073,17 +1072,16 @@
       create(:transaction, parent_activity: project, value: 100, report: report, date: Date.today)
       create(:transaction, parent_activity: project, value: 200, report: report, date: Date.today)
       create(:planned_disbursement, parent_activity: project, value: 1500, report: report, period_start_date: Date.today)
-      create(:planned_disbursement, parent_activity: project, value: 500, report: report, period_start_date: Date.today)
 
-      expect(project.variance_for_report_financial_quarter(report: report)).to eq(-1700)
+      expect(project.variance_for_report_financial_quarter(report: report)).to eq(-1200)
     end
   end
 
   describe "#forecasted_total_for_date_range" do
     it "returns the total of all the activity's planned disbursements scoped to a date range" do
       project = create(:project_activity, :with_report)
-      _disbursement_1 = create(:planned_disbursement, parent_activity: project, value: 200, period_start_date: Date.parse("13 April 2020"))
-      _disbursement_2 = create(:planned_disbursement, parent_activity: project, value: 1000, period_start_date: Date.parse("20 November 2020"))
+      _disbursement_1 = create(:planned_disbursement, parent_activity: project, value: 200, period_start_date: Date.parse("13 April 2020"), financial_quarter: 1, financial_year: 2020)
+      _disbursement_2 = create(:planned_disbursement, parent_activity: project, value: 1000, period_start_date: Date.parse("20 November 2020"), financial_quarter: 3, financial_year: 2020)
 
```

Comment:
Do these date ranges only work within the same financial quarter?

Reply:
This work is a bit odd, I should have been clearer in the PR message (apologies) at the moment the code that sorts the forecasts for display still relies on the `period_start_date` so we still have to include it, but this 'only one original' validation uses the financial quarter and year so for this 'interim' piece of work we need both - the work to fix the display and sorting is to come on this card: https://trello.com/c/Uw6hDK2r